pages/en/lb4/Using-components.md

```diff
@@ -35,4 +35,4 @@ In general, components can contribute the following items:
 In the future (before the GA release), components will be able to contribute additional items:
 
  - Models
- - [Repositories](Repositories.html)
+ - [Repositories](Repositories.html)
```

Comment:
Unrelated whitespace change, could you please revert?

Reply:
weird...I could not see where is the whitespace(difference), I copy paste the original file but diff is still here...

Reply:
Sounds like a character formatting problem (was this file made on Windows or something? :P)